app/src/main/res/layout/speed_select_dialog.xml

```diff
@@ -1,45 +1,51 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:padding="16dp"
-        android:orientation="vertical">
```

Comment:
Sorry for the reformatting, that's because you are the first person to change that file since we added the formatting check. Back when adding the check, I didn't want to re-format all files at once because that would have been a huge commit making it harder to use `git blame` and stuff

Reply:
No worries. It would be nice if there was an easy way to auto-format it according to the requirements but I didn't see anything in the contributor guide.

Reply:
I just run the CI formatter tool locally. In Android Studio, you can add it as an "external tool". Then you can reformat files using right-click.

https://github.com/ByteHamster/android-xml-formatter

Reply:
Cool, thanks. If I hit another file needing reformatting I'll try and set that up.